image/descriptor.go

```diff
@@ -49,6 +49,30 @@ func (d *descriptor) hash() string {
 	return pts[1]
 }
 
+func findDescriptors(w walker) (map[string]*descriptor, error) {
```

Comment:
Maybe `listReferences`?  In #159, the analogous function is [`refs.Engine.List()`][1].

[1]: https://github.com/opencontainers/image-spec/pull/159/files#diff-d8af84c58f6ae3369e0b5cbca06727dfR43